tests/args/regex/test_args.py

```diff
@@ -20,17 +20,17 @@
 
 
 @scenario_args('Every step takes a parameter with the same name')
-def test_steps():
+def _():
```

Comment:
in case of given you cen use it as a fixture later, right?

@given('I have a foo')
def foo():
    return 1

@then('Foo should be 1' )
def _(foo):
     assert foo == 1